src/Microsoft.CodeQuality.Analyzers.Exp/Core/Usage/DisposeMethodsShouldCallBaseClassDispose.cs

```diff
@@ -53,7 +53,7 @@ public override void Initialize(AnalysisContext context)
                 compilationContext.RegisterOperationBlockStartAction(operationBlockStartContext =>
                 {
                     if (!(operationBlockStartContext.OwningSymbol is IMethodSymbol containingMethod) ||
-                        !containingMethod.IsOverride ||
+                        containingMethod.OverriddenMethod == null ||
```

Comment:
at the test below, what overriden method is missing?

Reply:
It is missing `using System;` hence error case. I hit this while dogfooding the latest analyzer package.

Reply:
I will add a comment to added unit test.